src/css/main.css

```diff
@@ -1,4 +1,79 @@
-p {
-    color: red;
-    appearance: none;
+
+header {
+    background: #f0f0f0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+}
+
+.header img{
+    float: right;
+    padding-right: 16px;
+    padding-bottom: 20px;
+    padding-top: 20px;
+}
+
+.header h1 {
+    display: inline;
+    float: left;
+    color: black;
+    margin: 0%;
+    font-size: 24px;
+    padding-left: 16px;
+    padding-bottom: 20px;
+    padding-top: 20px;
+}
+
+article {
+    padding-top: 150px;
+}
+
+p.text {
+    color: black;
+    padding-left: 88px;
+    padding-right: 88px;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #C0C0C0;
+    width: 100%;
+}
+
+li {
+    float: left;
+}
+
+li a {
+    display: block;
+    color: black;
+    text-align: center;
+    padding: 16px;
+    text-decoration: none;
+}
+
+h2 {
+    text-align: left;
+    padding-left: 88px;
+    font-weight:bold;
+    font-size: 20px;
+}
+
+footer {
+    background: #C0C0C0;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    padding: 16px;
+}
+
+p.left{
```

Comment:
if you remove the `p` from this, then the remaining class would be what we call a `utility class`, which could be used whenever you want anything 'floated' left.  These are useful when combine classes and separating  'layout' from component 'design'.

the other thing with this is specificity. using a tag (`p`) is worth 10 points, the class (`.left`) is worth 1. The higher the specificity, the harder it is to override the styles - which will become a massive pain in the ass if you're not careful! so keeping specificity low is quite nice where possible (using an id is 100 points, which is why we generally stay well away from id's for styling!)

might be worth playing with this to see how it affects you. change the display style of the `.menu` to see if it takes affect. i.e.

```css
.menu {
   display: inline;
}
````

Reply:
Ahh! I didn't realise that was the order of how specificity worked. So with respect to the menu class, because i've styled ul { display:block} the specificity is higher and it won't override it.

I've added classes to the header and github image so that .left styles them as well as the text in the footer.